project.clj

```diff
@@ -59,7 +59,8 @@
    :repl { :plugins [[cider/cider-nrepl "0.10.0-SNAPSHOT"]]
            :repl-options {:init (set! *print-length* 500)}}             
    :dev {
-      :dependencies [[org.omcljs/om "0.9.0"]
+      :dependencies [;[org.omcljs/om "0.9.0"]
```

Comment:
I pointed out because I though it was accidental. I really don't mind if it's there or not.